src/sentry/static/sentry/app/components/dropdownReact.jsx

```diff
@@ -98,6 +104,15 @@ class DropdownReact extends React.Component {
     }
   };
 
+  // Decide whether dropdown should be closed when mouse leaves element
+  handleMouseLeave = e => {
+    const toElement = e.toElement || e.relatedTarget;
+
+    if (this.dropdownMenu && !this.dropdownMenu.contains(toElement)) {
```

Comment:
this is nice :+1:. possibly fragile but i'm not sure how else to go about it

Reply:
Should be fine